python/packages/nisar/products/writers/GcovWriter.py

```diff
@@ -802,6 +808,37 @@ def populate_processing_information(self):
             f'{parameters_group}/geo2rdr/deltaRange',
             1.0e-8)
 
+    def populate_processing_information_timing_corrections(self):
+        """
+        Populate the `processingInformation/timingCorrections` group of the
+        GCOV product
+        """
+        timing_corrections_group_path = \
+            ('science/LSAR/GCOV/metadata/processingInformation/'
```

Comment:
LSAR is hard-coded here. Is there a way to make this generic so it can be used to process S-SAR data?